packages/components/nodes/prompts/PromptLangfuse/PromptLangfuse.ts

```diff
@@ -0,0 +1,77 @@
+import { ICommonObject, INode, INodeData, INodeParams, PromptTemplate } from '../../../src/Interface'
+import { getBaseClasses, getInputVariables } from '../../../src/utils'
+import { PromptTemplateInput } from '@langchain/core/prompts'
+import { Langfuse } from "langfuse";
+
+class PromptLangfuse_Prompts implements INode {
+    label: string
+    name: string
+    version: number
+    description: string
+    type: string
+    icon: string
+    category: string
+    baseClasses: string[]
+    inputs: INodeParams[]
+
+    constructor() {
+        this.label = 'LangFuse Prompt Template'
+        this.name = 'promptLangFuse'
+        this.version = 1.0
+        this.type = 'PromptTemplate'
+        this.icon = 'prompt.svg'
+        this.category = 'Prompts'
+        this.description = 'Fetch schema from LangFuse to represent a prompt for an LLM'
+        this.baseClasses = [...getBaseClasses(PromptTemplate)]
```

Comment:
Hey! thanks for the contribution!

I've just added this [PR](https://github.com/FlowiseAI/Flowise/pull/2902) that allow users to add nodes created by community.

Can you add this `this.author = 'your name'` to here, so as we can differentiate that this is a node created by community, and will be shown on UI as well

Reply:
an example guide: https://docs.flowiseai.com/contributing/building-node

Reply:
Done!